### PR TITLE
Late July pull

### LINKED ISSUE #1803: Author of 'ADgk Mobile Advertising Rules ' and 'Baidu Extreme Purification' Relocated Repository to GitHub, Abandons Gitee
The Gitee repository will no longer be updated since July 21 and the subscribers should update their links promptly, the author said.

https://gitee.com/banbendalao/adguard

It thus results in changes to the two filter lists' links on filterlists.com.

For 'ADgk Mobile Advertising Rules - adgk手机去广告规则' 

Subscription:
abp:subscribe?location=https%3A%2F%2Fraw.githubusercontent.com%2Fbanbendalao%2FADgk%2Fraw%2FADgk.txt&amp;title=ADgk%20Mobile%20Advertising%20Rules

View:
https://raw.githubusercontent.com/banbendalao/ADgk/master/ADgk.txt

Home:
https://github.com/banbendalao/ADgk

For 'Baidu Extreme Purification - 百度搜索结果极限净化'

Subscription:
abp:subscribe?location=https%3A%2F%2Fraw.githubusercontent.com%2Fbanbendalao%2FADgk%2Fmaster%2Fkill-baidu-ad.txt&amp;title=Baidu%20Extreme%20Purification

View:
https://raw.githubusercontent.com/banbendalao/ADgk/master/kill-baidu-ad.txt

Home:
https://github.com/banbendalao/ADgk

Note: The author also provides alternative links for subscribers in countries/regions where connection to 'raw.githubusercontent.com' is blocked or very slow. 

https://github.com/banbendalao/ADgk/commit/2a4b105e5f5711ec24bf62cc5e95fe0dc1a02406

### PR DESCRIPTION
**Pull request by DandelionSprout**
* Added 2 lists.
* Removed 2 lists.
* Added mirrors for anti-AD and ADgk.
* Updated various descriptions, list titles, and languages.
* Added 1 software and syntax.
* Fix #1803.

### CODE REVIEW

**Comment by DandelionSprout** [comment]
I can't say I understand much of the error message I encountered this time around.

File: data/FilterList.json [hunk 1]
```diff
@@ -16968,7 +16968,8 @@
     "licenseId": 5,
     "name": "Synology's own",
     "syntaxId": 2,
-    "viewUrl": "https://in0de.synology.me/own.txt"
+    "viewUrl": "https://in0de.synology.me/own.txt",
+    "isDeleted": true,
```
Comment (collinbarrett) [inline data/FilterList.json:16972]:
```suggestion
    "isDeleted": true
```

Comment (collinbarrett) [reply data/FilterList.json:16972]: I think it's the trailing comma at the end of the json object. Debugged by looking at the fairly new "ling json" step [here](https://dev.azure.com/collinbarrett/FilterLists/_build/results?buildId=4051&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=dd349f25-7095-58cb-1ea1-12e8abf37d4a) and validating that the json spec doesn't allow trailing commas [here](https://stackoverflow.com/a/201856/2343739).

Comment (DandelionSprout) [reply data/FilterList.json:16972]: Well spotted. I hope I've taken care of it now.

**Pull request merged**